server/models/models.go

```diff
@@ -244,8 +244,8 @@ type WebhookRequest struct {
 }
 
 type Subscription struct {
-	Name       string `json:"name" bson:"name" valid:"required~please provide a valid subscription name"`
-	Type       string `json:"type" bson:"type" valid:"required~please provide a valid subscription type"`
+	Name string `json:"name" bson:"name" valid:"required~please provide a valid subscription name"`
+	//Type       string `json:"type" bson:"type" valid:"required~please provide a valid subscription type"`
```

Comment:
Are we taking this field out?

Reply:
Yeah, it's not being used. We don't need it since subscriptions are already scoped to a project which has the type already